examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.current = 0
+
+    this.context.questions_and_answers =
+      [
+        {
+          question: 'what popular tool for developing the client side application should we use?',
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'Bootstrap', 'jQuery'],
+          // answer: undefined // There is no answer hehe
+        },
+
+        {
+          question: 'What city is known as "The Big 🍎 Apple ?"',
+          answers: ['Seattle, WA', 'Skokie, IL', 'New York City'],
+          answer: 2
+        }
+      ]
+  }
+
+  onidle () {
+    const
+      previous = this.select
+        `footer button:first-of-type`
+
+    , next = this.select
+        `footer button:last-of-type`
+
+    , beginning =
+        this.context.current == 0
+
+    , end =
+        this.context.current ==
+          (this.context.questions_and_answers.length -1)
+
+    this.enable (beginning, previous)
+    this.enable (end, next)
+  }
+
+  // helpers
+  enable (predicate, element) {
+    predicate
+      ? element.disabled = true
+      : element.disabled = false
+
+  }
+
+  // handlers
+  onprevious ()
+    { this.context.current -- }
```

Comment:
So i don't miss it glancing over it @pachonk. More akin to the way Brendan E. stated how he would have made JS syntax. More akin to LISP/Scheme. That is where most of my spacing strategies are coming from. What what Brendan have written mixed with some readability for my weak eye. :-)